libraries/botbuilder-dialogs-declarative/src/resources/resourceExplorer.ts

```diff
@@ -43,6 +43,9 @@ export class ResourceExplorer {
      * @param {ResourceExplorerOptions} options The configuration options.
      */
     public constructor(options?: ResourceExplorerOptions);
+    /**
+     * @param providersOrOptions The list of [ResourceProvider](botbuilder-dialogs-declarative.ResourceProvider) or configuration options to initialize the current instance.
```

Comment:
I think this won't work to resolve the class.
```suggestion
     * @param providersOrOptions The list of [ResourceProvider](xref:botbuilder-dialogs-declarative.ResourceProvider) or configuration options to initialize the current instance.
```